<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWindow">
   <widget class="QGroupBox" name="matrixBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>A</string>
    </property>
    <widget class="QTableWidget" name="Matrix_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>211</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>22</number>
     </attribute>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="buildMatrixButton_1">
        <property name="text">
         <string>Построить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxY_matrix_1">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QSpinBox" name="spinBoxX_matrix_1">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="clearButton1">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="matrixBox2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>B</string>
    </property>
    <widget class="QTableWidget" name="Matrix_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QPushButton" name="buildMatrixButton_2">
        <property name="text">
         <string>Построить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxY_matrix_2">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QSpinBox" name="spinBoxX_matrix_2">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="clearButton2">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="toRight">
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="toLeft">
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="outputMatrixBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Результат</string>
    </property>
    <widget class="QTableWidget" name="Matrix_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>351</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="outputLine">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>560</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>191</width>
      <height>174</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QPushButton" name="matrixButon_convert_1">
       <property name="text">
        <string>Обратить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="matrixButon_transpose_1">
       <property name="text">
        <string>Транспонировать</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="matrix_line_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="matrix_operations_1">
       <property name="currentText">
        <string>Умножить на</string>
       </property>
       <item>
        <property name="text">
         <string>Умножить на</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Поделить на</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="matrix_calc_1">
       <property name="text">
        <string>Расчитать</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="matrixButon_determinant_1">
       <property name="text">
        <string>Определитель</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="result_to_A">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат -&gt; A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="result_to_B">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>520</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат -&gt; B</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QComboBox" name="optionsBox">
       <property name="currentText">
        <string>Сложить</string>
       </property>
       <item>
        <property name="text">
         <string>Сложить</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Вычесть</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Умножить</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="calculateButton">
       <property name="text">
        <string>Расчитать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>191</width>
      <height>174</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="2" column="1">
      <widget class="QPushButton" name="matrixButon_determinant_2">
       <property name="text">
        <string>Определитель</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="matrixButon_convert_2">
       <property name="text">
        <string>Обратить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="matrixButon_transpose_2">
       <property name="text">
        <string>Транспонировать</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="matrix_operations_2">
       <property name="currentText">
        <string>Умножить на</string>
       </property>
       <item>
        <property name="text">
         <string>Умножить на</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Поделить на</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="matrix_calc_2">
       <property name="text">
        <string>Расчитать</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="matrix_line_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
